stdlib/public/core/StringBridge.swift

```diff
@@ -148,11 +160,27 @@ internal enum _KnownCocoaString {
     }
 #endif
     
-    switch _unsafeAddressOfCocoaStringClass(str) {
+    /*WARNING EEK DANGER BEWARE
+      The following code does not support non-pointer isa. This is intentional!
+      But it's probably not what you wanted if you're looking for something to
+      copy. It also hard-codes the name of two private classes. Don't do that
+      either!
+      */
+    let unsafeClass = withUnsafePointer(to: str) { strPtr in
```

Comment:
You've got an extra level of indirection here, I believe. :-/

Reply:
oh, dammit. That's what I get for trying to not unsafeBitCast everything